src/@batch-flask/ui/editor/monaco-languages/json/pool.template.json

```diff
@@ -44,11 +44,11 @@
           "title": "The desired number of dedicated compute nodes in the pool.",
           "description": "The desired number of dedicated compute nodes in the pool."
         },
-        "targetLowPriorityNodes": {
+        "targetSpotLowPriorityNodes": {
           "type": "integer",
           "format": "int32",
-          "title": "The desired number of low-priority compute nodes in the pool.",
-          "description": "The desired number of low-priority compute nodes in the pool."
+          "title": "The desired number of spot/low-priority compute nodes in the pool.",
```

Comment:
Per feedback from Peter and Doug, please use "Spot/low-priority" everywhere to be consistent with Portal. See discussion in "Spot v-team sync" team channel for details.

Reply:
Fixed